src/ResXManager.Translators/OpenAITranslator.cs

```diff
@@ -331,6 +346,7 @@ private static IList<ICredentialItem> GetCredentials()
         return
         [
             new CredentialItem("AuthenticationKey", "Key"),
+            new CredentialItem("Url", "Endpoint Url", false) { Value = "https://api.openai.com" },
```

Comment:
Can you move the new URL item to the bottom, so it does not corrupt existing configurations?